### ISSUE TITLE
[Issue]: Ogg Vorbis music files make Jellyfin show pictures of the Dutch keyboardist Jacques Ogg(?)

### ISSUE THREAD

**Issue by vext01** [jellyfin/jellyfin#9406]
### Please describe your bug

a.k.a  Jacques Ogg has taken over my media library!

I installed jellyfin using the docker instructions from the website, pointed at my music collection, and it works great, with just one small quirk. Jacques Ogg (a Dutch keyboard player who I'd never heard of prior to this incident) is **everywhere**.

![image](https://user-images.githubusercontent.com/604955/221438465-3418fd08-ddb7-477c-81cb-341ec81767be.png)

![image](https://user-images.githubusercontent.com/604955/221438502-747c6747-9708-4014-85e8-6e3a2f58b0b2.png)

(I found out who it was using a reverse image search. He's a happy enough looking fellow, and it is a very nice picture of him!)

My music collection is about 90% Ogg Vorbis files, so I'd hazard a guess that the album/artist art finder is using the `.ogg` extension of the filenames. If that's the case, might it be an idea to strip the file extension before conducting the search?

### Jellyfin Version

Other

### if other:

10.8.9

### Environment

```markdown
- OS:official docker image under alpine linux
- Virtualization:docker
- Clients: problem persists on web, android and google tv.
- Browser: firefox
- FFmpeg Version:?
- Playback Method: ?
- Hardware Acceleration: ?
- Plugins: none
- Reverse Proxy: no
- Base URL: local IP address
- Networking: yes?
- Storage: volume from docker host.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vext01** [comment]
Album and arist image downloading is using musicbrainz by the way. I'm rescanning the library using The Audio DB now, will report back.

**Comment by vext01** [comment]
OK, so before I was talking about the `Metadata downloaders (Music Artists)` and `Metadata downloaders (Music Albums)`  in the library settings. Those don't affect anything.

Turning off "The Audio DB" in `Image fetchers (Music Artists)` and `Image fetchers (Music Albums)` means that Jacques no longer appears, but it obviously means that there are missing images for albums, and the artist image defaults to the album image, if present.

So this is a bug in the "The Audio DB" image fetcher then.

**Comment by vext01** [comment]
Looks like Jacques is here to stay.

**Comment by cvium** [comment]
> Looks like Jacques is here to stay.

I can't actually reproduce it. Gonna need more info about how you set up your music library, directory structure, filenames etc.

**Comment by vext01** [comment]
Can I email you the output of 'find' on my music collection?

**Comment by felix920506** [comment]
Please just post it here, thanks.

**Comment by TorchedSammy** [comment]
Jellyfin doesn't handle Ogg files properly. I wanted to move from my MP3 music library to opus, which is contained in Ogg, and it treats my music library (**that I have set as Music content type**) as videos???

![](https://safe.saya.moe/KVGE4TDWmAIi.png)

You can just take a small amount of music in FLAC, convert it to opus (or vorbis) and add a Music library in Jellyfin. This is an easy to reproduce issue.

**Comment by auspex** [comment]
I'm seeing the same problem as TorchedSammy

**Comment by felix920506** [comment]
I imported a library of 376 ogg vorbis tracks into my library on master. No signs of Jacques Ogg taking over my library

**Comment by felix920506** [comment]
@TorchedSammy @auspex The problem you are facing is a separate issue and should be reported separately instead of over here. Please open new issue reports.

**Comment by ShadowJonathan** [comment]
Not stale

https://nostalebots.xyz

**Comment by felix920506** [comment]
Yeah unfortunately this issue is going nowhere without additional info and/or a way to concretely reproduce it. 
Do any of you who are having issues have music files without metadata tags?
